delta/sourcing-psql/src/main/scala/ch/epfl/bluebrain/nexus/delta/sourcing/state/GlobalStateStore.scala

```diff
@@ -122,21 +110,19 @@ object GlobalStateStore {
         .option
         .transact(xas.read)
 
-    private def states(offset: Offset, strategy: RefreshStrategy): EnvelopeStream[S] =
-      StreamingQuery[Envelope[S]](
+    private def states(offset: Offset, strategy: RefreshStrategy): Stream[IO, S] =
+      StreamingQuery[GlobalStateValue[S]](
```

Comment:
Maybe you could use a tuple (S, Offset) here ?
Or make `GlobalStateValue`private ?

Reply:
Thanks, less code!